### PR TITLE
[FIX] web: trackback when formatFloat value is undefined

### PR DESCRIPTION
**Pull request by kais-odoo**
steps to produce:
- when measure field value set null if value is zero
- try to open cohort view measure
- trackback occurs 'value.toFixed is not a function'

Fix:
This PR, fixes the trackback when the measure value is null or undefined. It occurs when the measure field value is null in a particular condition.

task: 3502869

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by MissingNoShiny — APPROVED** [review]
LGTM, but with this, the checks on `value` in `odoo/addons/web/static/src/views/fields/formatters.js` seem redundant, so maybe we should remove them?

**Comment by MissingNoShiny** [comment]
Now that I think about it, the framework team will probably want this to be covered by tests, could you also add a test case for this? :innocent:

File: addons/web/static/src/core/utils/numbers.js [hunk 1]
```diff
@@ -177,6 +177,9 @@ export function humanNumber(number, options = { decimals: 0, minDigits: 1 }) {
  * @returns {string}
  */
 export function formatFloat(value, options = {}) {
+    if (value === false) {
```
Comment (aab-odoo) [inline addons/web/static/src/core/utils/numbers.js:180]: it makes no sense for a basic formatFloat (in utils/numbers) function to handle the `false` case. So I would use formatFloat from formatters.js in cohort if the value can be false.

**Review by auon-odoo — APPROVED** [review]

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work. 🙂 

robodoo r+

**Pull request closed without merging**